src/tests/JIT/IL_Conformance/Old/Conformance_Base/beq_r4.ilproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RestorePackages>true</RestorePackages>
-    <CLRTestPriority>1</CLRTestPriority>
+    <CLRTestPriority>0</CLRTestPriority>
```

Comment:
are you going to revert test changes before merge or want to keep them in pri0?

Reply:
No we need this extra test coverage